docker-compose.alfresco.yml

```diff
@@ -0,0 +1,91 @@
+version: '3.4'
+
+services:
+  db:
+    image: mdillon/postgis
+    # environment:
+    #   - POSTGRES_USER=${PG_USER:openzaak}
+    #   - POSTGRES_PASSWORD=${PG_PASSWORD:openzaak}
+
+  redis:
+    image: redis
+
+  web:
+    build: .
+    image: openzaak/open-zaak
+    environment:
+      - DJANGO_SETTINGS_MODULE=openzaak.conf.docker
+      - SECRET_KEY=${SECRET_KEY:-7(h1r2hk)8z9+05edulo_3qzymwbo&c24=)qz7+_@3&2sp=u%i}
+      - IS_HTTPS=no
+      - ALLOWED_HOSTS=*
+      - CACHE_DEFAULT=redis:6379/0
+      - CACHE_AXES=redis:6379/0
+      - SUBPATH=${SUBPATH:-/}
+      - CMIS_ENABLED=True
+      - CMIS_MAPPER_FILE=/app/config/cmis_mapper.json
+      - CMIS_BINDING=BROWSER
+    ports:
+      - 8001:8000
+    depends_on:
+      - db
+
+  nginx:
+    image: nginx
+    volumes:
+      - ./default.conf:/etc/nginx/conf.d/default.conf
+    ports:
+      - "9001:80"
+    depends_on:
+      - web
```

Comment:
Can we leave these services out (or only include the overrides from `docker-compose.yml`) and change the documentation to use `docker-compose -f docker-compose.yml -f docker-compose.alfresco.yml up` if you want to use the Aflresco-backed CMIS store?

This reduces the maintenance overhead in the docker-compose files because of the duplication of services/configuration.

Reply:
I will check if this works.

Reply:
It should be possible. The host to use for connections from Openzaak to Alfresco would be `host.docker.internal`. The two stacks would not be in the same docker network . Thats why that host is required. The port would be `11232` since that is the external port. It would not work with the internal port using this host.

The other method would be to manually create a network and make the docker compose files use this network. But the downside of that would be the manual creation of the network.

Reply:
@sergei-maertens Is [this](https://open-zaak.readthedocs.io/en/latest/development/getting_started.html?highlight=docker#testing-with-cmis-adapter-enabled) the correct documentation to adjust?

Reply:
`host.docker.internal` is a macOS thing (and possibly Windows) because Docker runs in a VM there, it doesn't work on Linux. Applying both docker files should use the same docker network, and you can extend the services by re-using the same service name (which can be used for the internal DNS in the bridge network still).

w/r to the referenced documentation, that's not the one - that should _only_ spin up Alfresco without Open Zaak services (because we run Open Zaak outside of Docker while actively developing/running tests).

The `docker-compose.yml` might just not be documented at all - or it would probably be in the README. I'm okay with documenting it with some comments at the top in the `docker-compose.yml` file - it's aimed at developers anyway.